test/native-tests/test-pal/sample.cpp

```diff
@@ -0,0 +1,70 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "pal.h"
+
+int wscmp(const char16_t *left, const char16_t *right)
```

Comment:
you're right, for some reason I thought this was removed from our copy.  I'll fix...